activity/actreturn/README.md

```diff
@@ -0,0 +1,50 @@
+# tibco-reply
```

Comment:
Should this be tibco-return?

Reply:
I would also take act out of actreturn and actreply. Since they are already inside activity folder it is a bit redundant